hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -225,3 +282,86 @@ rpi-eeprom-config --out pieeprom-netboot.bin --config bootconf.txt pieeprom.bin
+
+## BETA setup instructions
+These instructions assume that you are familiar with manual firmware and bootloader updates and understand how to revert to a previous version if you want to revert the changes. If not, please wait until the features are available in a full Raspbian release image.
+
+There will be a post on the Raspberry Pi [General Discussion forum](https://www.raspberrypi.org/forums/viewforum.php?f=63) when the binaries are available.
+
+### Update the bootloader
+```
+# From a standard Raspbian SD card boot
+sudo apt update
+sudo apt full-upgrade
+
+# As root edit /etc/default/rpi-eeprom-update and select BETA releases
+
+# Install the BETA version of the bootloader and replace the current configuration settings to enable USB boot.
+# See BOOT_ORDER property if you wish to migrate the configuration by hand.
+sudo rpi-eeprom-update -d -f /lib/firmware/raspberrypi/bootloader/beta/pieeprom-2020-05-15.bin
+
+# Reboot and check the bootloader version 
+vcgencmd bootloader_version 
+vcgencmd bootloader_config
+```
+
+### Create a bootable USB drive
+* Use the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to flash Raspbian to a USB mass storage device. Other distros have not been tested and may require updates (e.g. u-boot). One reason for having a public beta is to help get USB MSD boot support into other distros.
+* Download the updated firmware files `*.elf *.dat` from the `msd-boot` branch of the [Raspberry Pi Firmware](https://github.com/raspberrypi/firmware) Github repo. 
+* Alternatively use `rpi-update branch=msd-boot` to update the firmware on a Raspbian SD card install and copy the files from there.
+* Copy these updates to the boot partition on the USB device. From now on `rpi-update branch=msd-boot` can be used from within Raspbian on the USB boot device.
+* A Linux kernel updates is not required. Raspbian has been tested using the 4.19 and 5.4 (32 and 64 bit) kernel.
+
+### USB device compatiblity
+This no explicit set of supported devices. Initially we recommend using a USB pen drive or SSD. Hard drives will probably require a powered HUB and in all cases you should verify that the devices work correctly from within Raspbian using an SD card boot.
+
+Please post interoperability reports (positive or negative) on the Raspberry Pi forums. 
+
+### USB_MSD_EXCLUDE_VID_PID
+A list of up to 4 VID/PID pairs specifying devices which the bootloader should ignore. If this matches a HUB then the HUB won’t be enumerated causing all downstream devices to be excluded.
+This is intended to allow problematic (e.g. very slow to enumerate) devices to be ignored during boot enumeration. This is specific to the bootloader and is not passed to the OS.
+
+The format is a comma separated list of hexadecimal values with the VID as most significant nibble. Spaces are not allowed.
+E.g. `0347000a0,0a4231234`
```

Comment:
How come these are both 9 digits long - should they be 8 digits long?

Reply:
typo, although it would work

Reply:
typo, it will work, sort of